wallet-new/integration/RandomStateWalk.hs

```diff
@@ -752,12 +751,6 @@ initialWalletState wc = do
   where
     fromResp = (either throwM (pure . wrData) =<<)
 
-log :: MonadIO m => Text -> m ()
-log = putStrLn . mappend "[TEST-LOG] "
-
-ppShowT :: Show a => a -> Text
-ppShowT = fromString . ppShow
-
```

Comment:
This has nothing to do with the PR. 
Once again: Avoid cosmetic changes, and if some, please have them in a separated PR or *at the very least*, in a separated commit which a clear commit message stating the reason of the change.

Reply:
I was assuming that you made a general comment that does not require changes to the code.
Therefore I clicked on the resolved button.
I'm happy to make changes, discuss the code and make improvements
and there are definitely ways to improve this PR.
But for me there was no clear way how to change the code in reaction to your comment.
Why is deleting something is a cosmetic change?

I deleted those two lines because I was moving code from TransactionsSpecs to Utils.hs,
which brought a second definition of `ppShowT` into scope which made the use `ppShowT` in RandomStateWalk ambiguous.
To make it compile again I had to remove the code duplication
that was present in the code before.
When I prepared the PR I made tons of changes all to my best knowledge and with the only intent to improve the code.
This PR in not perfect and it is not polished to the last character.
But IMHO there is a trade offs and for me the PR seems to be perfectly acceptable.
Developers should tread each other with respect and trust.
I have deleted those two lines of code and it is fair to assume that I had reason for that change.

It took about 10 times longer now to discuss that change than to make the edits itself.